### ISSUE TITLE
"Error clearing seed" printed out in console

### ISSUE THREAD

**Issue by danielschroder** [kotest/kotest#3304]
**Which version of Kotest are you using**
5.5.3

I'm just getting started with property testing and I'm having a very strange behavior when I change the `name` of the `context` that my `checkAll` function is running in.

I'm trying to name my contexts based on the functions/methods it's testing so, I wrote the following code:

```kotlin
context("CodedCompanion.fromCodeOrNull()") {
    context("Returns the correct object with valid codes") {
        withData(
            nameFn = { "Code ${it.first} returns an object with code ${it.second.code}" },
            0u to CodedTestA,
            1u to CodedTestB,
            2u to CodedTestC
        ) {
            CodedTest.fromCodeOrNull(it.first) shouldBeSameInstanceAs it.second
        }
    }

    test("Returns null with invalid codes") {
        checkAll(Arb.uInt(min = 3u)) {
            CodedTest.fromCodeOrNull(it) shouldBe null
        }
    }
}
```

OK. This works as expected but, when I add the parameter type to that first context name, the console starts printing out "Error clearing seed".

Code with the print log:

```kotlin
context("CodedCompanion.fromCodeOrNull(code: T)") {
    context("Returns the correct object with valid codes") {
        withData(
            nameFn = { "Code ${it.first} returns an object with code ${it.second.code}" },
            0u to CodedTestA,
            1u to CodedTestB,
            2u to CodedTestC
        ) {
            CodedTest.fromCodeOrNull(it.first) shouldBeSameInstanceAs it.second
        }
    }

    test("Returns null with invalid codes") {
        checkAll(Arb.uInt(min = 3u)) {
            CodedTest.fromCodeOrNull(it) shouldBe null
        }
    }
}
```

It's exactly the same code as the first one, except that now the first line has the parameter inside de parenthesis.

The tests are all successful. But that error log is very strange.

**Comment by sksamuel** [comment]
This will be because it uses the test name for the file name for the seed, and I guess it doesn't like a character in the 2nd example.

**Comment by sksamuel** [comment]
@danielschroder are you on windows?

**Comment by danielschroder** [comment]
@sksamuel Yes. The issue occurred while running the tests on a Windows machine.

**Comment by sksamuel** [comment]
The fix in place for 5.5.5 should fix it then.
